app/api/v1/endpoints/export_data.py

```
@@ -0,0 +1,367 @@
+    **Export Types:**
+    - `full`: Export all user data (journals, entries, media, settings)
+    - `journal`: Export specific journals (requires journal_ids)
+    - `admin_full`: Export all users' data (admin only)
+    """
+    try:
+        export_type = ExportType(export_request.export_type)
+    except ValueError:
+        raise HTTPException(
+            status_code=400,
+            detail=f"Invalid export type: {export_request.export_type}"
+        )
```

Ruff (0.14.4)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)